src/shared/models/global.models.ts

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
I love this namespaced-way of defining types.  Is the philosophy that only reused types are here?  I was noticing types defined in a lot of other places besides here.